scripts/install

```diff
@@ -38,76 +42,76 @@ grep -q -R 'jessie-backports' /etc/apt/sources.list{,.d} || {
 }
 
 # Install dependencies
-ynh_package_install_from_equivs ../conf/${DEPS_PKG_NAME}.control \
-  || ynh_die "Unable to install dependencies"
+ynh_package_install_from_equivs ../conf/${DEPS_PKG_NAME}.control
```

Comment:
Hum ... I have a problem with remove script. If i use ynh_package_autoremove helper, i'm need to add all package also ?

Example
```
if ynh_package_is_installed php5-cli php5-common php5-intl php5-json php5-mcrypt \
                              php-pear php-auth-sasl php-mail-mime php-patchwork-utf8 \
                              php-net-smtp php-net-socket php-crypt-gpg \
                              php-net-ldap2 php-net-ldap3; then
  ynh_package_autoremove php5-cli php5-common php5-intl php5-json php5-mcrypt \
                              php-pear php-auth-sasl php-mail-mime php-patchwork-utf8 \
                              php-net-smtp php-net-socket php-crypt-gpg \
                              php-net-ldap2 php-net-ldap3
fi
```